test/AutoDiff/downstream/inout_parameters.swift

```diff
@@ -0,0 +1,128 @@
+// RUN: %target-run-simple-swift
+// REQUIRES: executable_test
+
+// `inout` parameter differentiation tests.
+
+import StdlibUnittest
+
+var InoutParametersTests = TestSuite("InoutParameters")
+
+// TODO: Move floating-point mutating operation tests to
```

Comment:
This is tracked by TF-1173.
```suggestion
// TODO(TF-1173): Move floating-point mutating operation tests to
```